src/Validation/Validator.php

```diff
@@ -160,14 +160,26 @@ public function __construct()
     }
 
     /**
-     * Returns an array of fields that have failed validation. On the current model. This method will
-     * actually run validation rules over data, not just return the messages.
+     * Validates and returns an array of failed fields and their error messages.
      *
      * @param array $data The data to be checked for errors
      * @param bool $newRecord whether the data to be validated is new or to be updated.
-     * @return array Array of invalid fields
+     * @return array Array of failed fields
+     * @deprecated 3.9.0 Renamed to validate()
```

Comment:
`, and will remove in future`

Reply:
This is a method rename we could add a rector for to the 40 ruleset.